rosidl_adapter/rosidl_adapter/parser.py

```diff
@@ -427,7 +427,7 @@ def __str__(self):
 def parse_message_file(pkg_name, interface_filename):
     basename = os.path.basename(interface_filename)
     msg_name = os.path.splitext(basename)[0]
-    with open(interface_filename, 'r') as h:
+    with open(interface_filename, 'r', encoding='utf-8') as h:
```

Comment:
This seems fine to me. Just noting that this requires interface files to be utf-8, where they might have worked fine if they happened to be using the default system encoding before.